src/Windowing/OpenToolkit.Windowing.Common/Interfaces/INativeWindowProperties.cs

```diff
@@ -88,12 +88,12 @@ public interface INativeWindowProperties
         /// Gets or sets a <see cref="OpenToolkit.Mathematics.Vector2" /> structure that contains the location of this window on the
         /// desktop.
         /// </summary>
-        Vector2 Location { get; set; }
+        Vector2i Location { get; set; }
 
         /// <summary>
         /// Gets or sets a <see cref="OpenToolkit.Mathematics.Vector2" /> structure that contains the external size of this window.
         /// </summary>
-        Vector2 Size { get; set; }
+        Vector2i Size { get; set; }
```

Comment:
Isn't a `Box2i` more appropriate here?

Reply:
A size does not have a position, does it?
It's just the size, namely Width and Height. One could make a specific type for that, but as it would be structurally the same as Vector2i I don't think it is really necessary.

Reply:
I know, but `Box2i` is more strictly correct. Besides, I believe void intends to make this change anyway as this is marked as blocked by #918

Reply:
I size only does have two parameters, Box2i has 4, how would that work? I mean you could combine Location and Size into one Box2i, is that what you actually meant?
(I just opened the properties file to look for Location, where it is shown under your comment, which is not the case in the overview, so I assume that was a miscommunication)

Reply:
Box2 is a rectangle, no? whereas Vector2 is a point.

Reply:
Box2 is a rectangle on a 2D plane. Vector2 is a two-dimensional quantity.

Reply:
Exactly, so Box2 seems more applicable for the representation of a window size.

Reply:
"size" doesn't have a location :)

Reply:
Meh good point.

Reply:
The real pun wizard right here.

Reply:
Ohh so @Perksey didn't mean to merge Location and Size into Box2i?

Reply:
You could make the location and the size of the window into a combined value, no?

Reply:
@jvbsl I had nothing to do with Box2's surface API

Reply:
There was a reason why I used two vectors instead of a box, but I can't remember anymore...

Reply:
Oh, I remember now! It's because the boxes only store the coordinates of the points - to get the size, I'd have to subtract the top-left point from the bottom-right one, and send a new Vector2 back from that. I figured it was better to put the allocations on the Box2, because those were less likely to be used than the vectors.